ironfish/src/wallet/account/encryptedAccount.test.ts

```diff
@@ -0,0 +1,49 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { encrypt } from '@ironfish/rust-nodejs'
+import { useAccountFixture } from '../../testUtilities/fixtures/account'
+import { createNodeTest } from '../../testUtilities/nodeTest'
+import { AccountValueEncoding } from '../walletdb/accountValue'
+import { EncryptedAccount } from './encryptedAccount'
+
+describe('EncryptedAccount', () => {
+  const nodeTest = createNodeTest()
+
+  it('can decrypt an encrypted account', async () => {
+    const passphrase = 'foobarbaz'
+    const { node } = nodeTest
+    const account = await useAccountFixture(node.wallet)
+
+    const encoder = new AccountValueEncoding()
+    const data = encoder.serialize(account.serialize())
+
+    const encryptedData = encrypt(data, passphrase)
```

Comment:
Will be refactored when the decrypted account has an `encrypt` method for better encapsulation.